test/parallel/test-stream-base-prototype-accessors.js

```diff
@@ -24,4 +24,20 @@ const TTY = process.binding('tty_wrap').TTY;
   assert.throws(() => {
     TTY.prototype._externalStream;
   }, msg);
+
+  // Should not throw for Object.getOwnPropertyDescriptor
+  assert.strictEqual(
+    typeof (Object.getOwnPropertyDescriptor(TTY.prototype, 'bytesRead')),
```

Comment:
You don’t need the parentheses after `typeof` here

Reply:
👍 Removed.